content/sidebar/applicant_studentlife.md

```diff
@@ -0,0 +1,8 @@
+title: Добро пожаловать в мир ФМ
+slug: link_studentlife
```

Comment:
First, a `slug` should not contain `-`. Second, we name it the same as the file's name. So let it be `applicant-studentlife`.